Java/Marketing-API/MarketingAPISample

```diff
@@ -0,0 +1,243 @@
+            }
+        //create the one-line json      
+        contact.put("emailAddress", emailAddress);
+        json = gson.toJson(contact);
+            
+        //with the JSON object created and the authorization credentials created
+        //we can open the connection and set the address to unsubscribed
+        try{
+        URL myURL = new URL(serviceURL);
+        HttpURLConnection connection = (HttpURLConnection)myURL.openConnection();
+            connection.setRequestMethod("POST");
+            connection.setRequestProperty  ("Authorization","Basic "+encoding);
+            connection.setRequestProperty("Content-Type", "application/json");
+            connection.setUseCaches(false);
+            connection.setDoOutput(true);
+            connection.setDoInput(true);
+            connection.setAllowUserInteraction(false);
+            
+                OutputStream out = connection.getOutputStream();
+                OutputStreamWriter wr = null;
+                    try {
+                        wr = new OutputStreamWriter(out);
+                        wr.write(json);
+                            wr.close();
+                    } catch (IOException e) {
+                            e.printStackTrace();
+                    }
+                    
+                InputStream in = connection.getInputStream();
+                BufferedReader read = null;
+                    try{
+                        read = new BufferedReader(new InputStreamReader(in));
+                        line = read.readLine();
+                            read.close();
+                    } catch (IOException e){
+                        e.printStackTrace();
+                    }
+                    
+            connection.disconnect();
+        }catch(Exception e){
+            e.printStackTrace();
+        }
+        return line;
+    }
+    
+    private static String deleteContact(){
+        String emailAddress;
+        String serverId;
+        String APIKey;
+        String serviceURL;
+        HttpURLConnection connection;
+        String encoding = null;
+        String listId;
+        String line = null;
+        try{
+            emailAddress = "test@example.com";
+            serverId = "YOUR SERVER ID";
+            APIKey = "YOUR MARKETING AIP KEY";
+            listId = "LIST ID";//List IDs can be found using the getList method
+            serviceURL = "https://api.socketlabs.com/marketing/v1/lists/"+listId+"/contacts/?emailAddress="+emailAddress;
+            
+            //encode credentials
+            try{
+               encoding = Base64.getEncoder().encodeToString((serverId +":"+APIKey).getBytes("utf-8") );
+            } catch (Exception d){
+                d.printStackTrace();
+            }
+            //establish connection
+            URL myURL = new URL(serviceURL);
+            connection = (HttpURLConnection)myURL.openConnection();
+                connection.setRequestMethod("DELETE");
```

Comment:
100% understand.   Just trying to keep it consistent with the other code in the repository. :)